src/renderer/components/description/description.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './description.scss';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  selectCurrentUser,
+} from '../../../shared/slices/currentUserSlice';
+import {
+  selectDefaultDirPath,
+  setDefaultDirPath,
+} from '../../../shared/slices/configurationSlice';
+import findLocalPublications from '../../../main/node/file-manager';
+
+import {
+  installGatsbyCLI,
+  selectGatsbyInstallStatus,
+} from '../../../shared/slices/gatsbyInstallSlice';
+
+// TODO: delete exemplary buttons from here
+const Description = () => {
+  const currentUser = useSelector(selectCurrentUser);
+  const path = useSelector(selectDefaultDirPath);
+
+  // log user to showcase state changes if you don't have Redux Tools
+  console.log(currentUser);
+  console.log(path);
+  console.log(findLocalPublications());
+  const gatsbyInstallStuff = useSelector(selectGatsbyInstallStatus);
+
+  // log user to showcase state changes if you don't have Redux Tools
```

Comment:
why is this duplicated?

Reply:
I failed with merging my branch with main, because my branch had the descrption file while the main didn't have it, but I don't know how to delete the files nonexistent main files without deleting my changes

Reply:
Ok, just delete the duplicate